### PR TITLE
Removes the early classified announcement from the blob gamemode

### PR DESCRIPTION
**Pull request by Qwertytoforty**
##This does _not_ remove the level 5 bio hazard announcement.

## What Does This PR Do

Removes the classified announcement sent before the level 5 biohazard warning on blob rounds, and the paper that goes with it.
Also removes the declared variable, which does nothing.

## Why It's Good For The Game

Roundstart blob often has it hard, but often it is spotted quite fast. When it is not spotted fast, it has a larger issue.

image

This message will play. Without fail, even before anyone reads the paper, some crew will scream out blob, even if they do not see the paper and what it says. or the blob. This is silly, that crew instantly presume this message is a blob announcement. 
And of course, barring meta, this announcement can come really fast on a blob round.
Thus, this PR removes this announcement.

## Changelog
:cl:
tweak: Removes the early biohazard warning on blob rounds, but not the level 5 biohazard warning.
/:cl:

### CODE REVIEW

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]
image
Remove this too while you're at it

**Review by AffectedArc07 — APPROVED** [review]

**Comment by DogsParadise** [comment]
I was trying to counter act this problem as well, by creating a similar biohazard message with my mosquito pr

File: code/game/gamemodes/blob/blob.dm [hunk 1]
```diff
@@ -177,12 +176,8 @@ GLOBAL_LIST_EMPTY(blob_nodes)
...
 		burst_blobs()
 
-		// Stage 0
-		sleep(wait_time)
-		stage(0)
-
 		// Stage 1
-		sleep(wait_time)
+		sleep((wait_time * 2))
 		stage(1)
```
Comment (S34NW) [inline code/game/gamemodes/blob/blob.dm:181]:
```suggestion
		addtimer(CALLBACK(src, .proc/stage, 1), (wait_time * 2))
```

**Review by S34NW — COMMENTED** [review]
Could you also move the other `sleep` calls in this proc to `addtimer` if possible?

File: code/game/gamemodes/blob/blob.dm (hunk 1 above)
Comment (Qwertytoforty) [reply code/game/gamemodes/blob/blob.dm:181]: sure

File: code/game/gamemodes/blob/blob.dm [hunk 2]
```diff
@@ -75,8 +74,7 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 	log_game("[key_name(blob)] has been selected as a Blob")
 	greet_blob(blobmind)
 	to_chat(blob, "<span class='userdanger'>You feel very tired and bloated!  You don't have long before you burst!</span>")
-	spawn(600)
-		burst_blob(blobmind)
+	addtimer(CALLBACK(src, .proc/burst_blob, blobmind), 600)
```
Comment (S34NW) [inline code/game/gamemodes/blob/blob.dm:77]:
```suggestion
	addtimer(CALLBACK(src, .proc/burst_blob, blobmind), 60 SECONDS)
```

File: code/game/gamemodes/blob/blob.dm [hunk 3]
```diff
@@ -173,33 +170,22 @@ GLOBAL_LIST_EMPTY(blob_nodes)
...
 
 		// Stage 1
-		sleep(wait_time)
-		stage(1)
+		addtimer(CALLBACK(src, .proc/stage, 1), (wait_time * 2))
 
 		// Stage 2
-		sleep(30000)
-		stage(2)
+		addtimer(CALLBACK(src, .proc/stage, 2), 30000)
```
Comment (S34NW) [inline code/game/gamemodes/blob/blob.dm:179]:
```suggestion
		addtimer(CALLBACK(src, .proc/stage, 2), 50 MINUTES)
```

File: code/game/gamemodes/blob/blob.dm [hunk 4]
```diff
@@ -173,33 +170,22 @@ GLOBAL_LIST_EMPTY(blob_nodes)
...
 		show_message("<span class='userdanger'>You feel like you are about to burst.</span>")
 
-		sleep(wait_time / 2)
-
-		burst_blobs()
-
-		// Stage 0
-		sleep(wait_time)
-		stage(0)
+		addtimer(CALLBACK(src, .proc/burst_blobs), (wait_time * 2))
```
Comment (dearmochi) [inline code/game/gamemodes/blob/blob.dm:173]: Shouldn't it be `wait_time / 2`?

Comment (Qwertytoforty) [reply code/game/gamemodes/blob/blob.dm:173]: It should be, yeah.

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**